deployment/media-insights-stack.yaml

```diff
@@ -592,10 +592,32 @@ Resources:
         StreamViewType: "NEW_AND_OLD_IMAGES"
 
   # Services - S3
-  # TODO: Best Practice - S3 Bucket should have access logging configured, S3 Bucket should have encryption option set
+  DataplaneLogsBucket:
+    DeletionPolicy: Retain
```

Comment:
Should UpdateReplacePolicy also be Retain?

I did a quick compare to another solution to check if the S3 logs bucket is consistent with ours.  I noticed they have UpdateReplacePolicy set to Retain as well.  I guess for the case where there is a CF update that cause the bucket to be replaced.  A couple of actions do this: changing the bucket name and changing the object lock.

Reply:
I'm not sure if there is a case where that would occur, but certainly worth discussing at our next meeting.